src/providers/fontsource.ts

```diff
@@ -108,8 +108,18 @@ async function getFontDetails (family: string, variants: ResolveFontFacesOptions
   const fontFaceData: NormalizedFontFaceData[] = []
 
   for (const subset of subsets) {
-    for (const weight of weights) {
-      for (const style of styles) {
+    for (const style of styles) {
+      if (font.variable) {
+        fontFaceData.push({
+          style,
+          weight: [font.weights[0]!, font.weights.slice(-1)[0]!],
+          src: [
+            { url: `https://cdn.jsdelivr.net/fontsource/fonts/${font.id}:vf@latest/${subset}-wght-${style}.woff2`, format: "woff2" }
```

Comment:
Is it possible to get this URL from the API, by any chance?

Reply:
@danielroe Looks like they don't, so I just constructed a url. Sending a GET request from `https://api.fontsource.org/v1/fonts/inter` gives:

<details>
<summary>Response</summary>
<br>

```json
{
    "category": "sans-serif",
    "defSubset": "latin",
    "family": "Inter",
    "id": "inter",
    "lastModified": "2023-09-14",
    "license": "OFL-1.1",
    "npmVersion": "5.0.17",
    "source": "https://github.com/google/fonts",
    "styles": [
        "normal"
    ],
    "subsets": [
        "cyrillic",
        "cyrillic-ext",
        "greek",
        "greek-ext",
        "latin",
        "latin-ext",
        "vietnamese"
    ],
    "type": "google",
    "unicodeRange": {
        "cyrillic": "U+0301,U+0400-045F,U+0490-0491,U+04B0-04B1,U+2116",
        "cyrillic-ext": "U+0460-052F,U+1C80-1C88,U+20B4,U+2DE0-2DFF,U+A640-A69F,U+FE2E-FE2F",
        "greek": "U+0370-0377,U+037A-037F,U+0384-038A,U+038C,U+038E-03A1,U+03A3-03FF",
        "greek-ext": "U+1F00-1FFF",
        "latin": "U+0000-00FF,U+0131,U+0152-0153,U+02BB-02BC,U+02C6,U+02DA,U+02DC,U+0304,U+0308,U+0329,U+2000-206F,U+2074,U+20AC,U+2122,U+2191,U+2193,U+2212,U+2215,U+FEFF,U+FFFD",
        "latin-ext": "U+0100-02AF,U+0304,U+0308,U+0329,U+1E00-1E9F,U+1EF2-1EFF,U+2020,U+20A0-20AB,U+20AD-20C0,U+2113,U+2C60-2C7F,U+A720-A7FF",
        "vietnamese": "U+0102-0103,U+0110-0111,U+0128-0129,U+0168-0169,U+01A0-01A1,U+01AF-01B0,U+0300-0301,U+0303-0304,U+0308-0309,U+0323,U+0329,U+1EA0-1EF9,U+20AB"
    },
    "variable": true,
    "variants": {
        "100": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-100-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-100-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-100-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-100-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-100-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-100-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-100-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-100-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-100-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-100-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-100-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-100-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-100-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-100-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-100-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-100-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-100-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-100-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-100-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-100-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-100-normal.woff2"
                    }
                }
            }
        },
        "200": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-200-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-200-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-200-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-200-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-200-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-200-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-200-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-200-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-200-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-200-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-200-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-200-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-200-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-200-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-200-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-200-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-200-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-200-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-200-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-200-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-200-normal.woff2"
                    }
                }
            }
        },
        "300": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-300-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-300-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-300-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-300-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-300-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-300-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-300-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-300-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-300-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-300-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-300-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-300-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-300-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-300-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-300-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-300-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-300-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-300-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-300-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-300-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-300-normal.woff2"
                    }
                }
            }
        },
        "400": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-400-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-400-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-400-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-400-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-400-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-400-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-400-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-400-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-400-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-400-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-400-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-400-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-400-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-400-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-400-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-400-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-400-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-400-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-400-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-400-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-400-normal.woff2"
                    }
                }
            }
        },
        "500": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-500-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-500-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-500-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-500-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-500-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-500-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-500-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-500-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-500-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-500-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-500-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-500-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-500-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-500-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-500-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-500-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-500-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-500-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-500-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-500-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-500-normal.woff2"
                    }
                }
            }
        },
        "600": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-600-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-600-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-600-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-600-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-600-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-600-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-600-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-600-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-600-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-600-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-600-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-600-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-600-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-600-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-600-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-600-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-600-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-600-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-600-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-600-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-600-normal.woff2"
                    }
                }
            }
        },
        "700": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-700-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-700-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-700-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-700-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-700-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-700-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-700-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-700-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-700-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-700-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-700-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-700-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-700-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-700-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-700-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-700-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-700-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-700-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-700-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-700-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-700-normal.woff2"
                    }
                }
            }
        },
        "800": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-800-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-800-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-800-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-800-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-800-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-800-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-800-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-800-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-800-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-800-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-800-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-800-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-800-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-800-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-800-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-800-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-800-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-800-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-800-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-800-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-800-normal.woff2"
                    }
                }
            }
        },
        "900": {
            "normal": {
                "cyrillic": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-900-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-900-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-900-normal.woff2"
                    }
                },
                "cyrillic-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-900-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-900-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/cyrillic-ext-900-normal.woff2"
                    }
                },
                "greek": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-900-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-900-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-900-normal.woff2"
                    }
                },
                "greek-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-900-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-900-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/greek-ext-900-normal.woff2"
                    }
                },
                "latin": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-900-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-900-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-900-normal.woff2"
                    }
                },
                "latin-ext": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-900-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-900-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/latin-ext-900-normal.woff2"
                    }
                },
                "vietnamese": {
                    "url": {
                        "ttf": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-900-normal.ttf",
                        "woff": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-900-normal.woff",
                        "woff2": "https://cdn.jsdelivr.net/fontsource/fonts/inter@latest/vietnamese-900-normal.woff2"
                    }
                }
            }
        }
    },
    "version": "v13",
    "weights": [
        100,
        200,
        300,
        400,
        500,
        600,
        700,
        800,
        900
    ]
}
```

</details>